content/en/docs/KRE/user/50_krt_v2/specs/index.md

```diff
@@ -0,0 +1,121 @@
+entrypoint:
+  proto: public_input.proto
+  image: konstellation/kre-entrypoint:latest
+config:
+  variables:
+workflows:
+  - name: classificator
+    entrypoint: Classificator
+    exitpoint: exitpoint # name must match node's
+    nodes:
+      - name: etl
+        image: konstellation/kre-py:latest
+        src: src/etl/main.py
+        subscriptions:
+          - "entrypoint"
+
+      - name: email-classificator
+        image: konstellation/kre-py:latest
+        src: src/email_classificator/main.py
+        subscriptions:
+          - "etl"
+
+      - name: repairs-handler
+        image: konstellation/kre-go:latest
+        src: bin/repairs_handler
+        subscriptions:
+          - "email-classificator.repairs" #consume from a subtopic
+
+      - name: stats-storer
+        image: konstellation/kre-go:latest
+        src: bin/stats_storer
+        subscriptions:
+          - "email-classificator"
+
+      - name: exitpoint
+        image: konstellation/kre-go:latest
+        src: bin/exitpoint
+        subscriptions:
+          - "etl"
+          - "stats-storer"
+```
+
+## Fields Specs
+
+Here is a description of each field divided in five main concepts that make a KRT file:
+
+### Metadata
+
+Descriptive information of the Runtime Version.
+
+- **version**: the unique ID of this version. Must be in lowercase and can only contain letters, numbers and "-".
+- **krtVersion**: indicates the krt version this file is using. Only values allowed are "v1" and "v2".
+- **description**: brief text describing the content or functionality of this version.
+
+### Configuration
+
+This section describes all variables and files that are environment related and would be configured
+once the Runtime Version is uploaded to a KAI Server instance. This includes sensible data that should not
+be included on the KRT file, for example passwords.
+
+All configuration defined here is mandatory, the Runtime Version won't run if any of them is undefined.
+
+- **config**:
+  - **variables**: list of variable names that are going to be defined in KAI Server as environment variables. Must be in uppercase and can only contain letters, numbers and "_".
+  - **files**: list of file names.
+
+### Entrypoint
+
+This section describes how to access the Runtime Version.
+
+- **entrypoint**:
+  - **proto**: protobuffer file describing messages and services contained in this Runtime Version.
+  - **image**: base image and tag used to run the entrypoint service. It's provided by [Konstellation registry](https://hub.docker.com/u/konstellation).
+
+### Workflows
+
+This section includes the list of workflows contained in the KRT file. Each workflow has one or more nodes defined inside, of which one of them must be assigned to be the exitpoint.
+
+- **workflows** (a list of):
+  - **name**: an identifier text, must be unique in the workflow list.
+  - **entrypoint**: the name of a serviced defined in the proto buffer file of the entrypoint. See `entrypoint.proto` above.
```

Comment:
Here you are referencing to an `entrypoint.proto`, but I cannot find it.
Is it missing?

If not, could we add a direct link to it hereto make the navigation easier?

Reply:
They link was missing yes, now it should be fine